### ISSUE TITLE
far jump 0x33 crash

### ISSUE THREAD

**Issue by pinpiin** [DynamoRIO/dynamorio#3326]
Hi!

Is DR support instrumentation of application that use far jump 0x33?

**Comment by derekbruening** [comment]
Yes, see suite/tests/win32/mixedmode.c for a test.  If there is a specific problem, please provide details; if this is a general question, please use the users list rather than the issue tracker.
